models/models.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#include <pybind11/pytypes.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/numpy.h>
+#include <boost/graph/adjacency_list.hpp>
+#include <boost/graph/labeled_graph.hpp>
+#include <boost/graph/breadth_first_search.hpp>
+#include <boost/pending/indirect_cmp.hpp>
+#include <boost/range/irange.hpp>
+#include <boost/archive/text_iarchive.hpp>
+#include <boost/archive/text_oarchive.hpp>
+#include <boost/variant.hpp>
+
+#include <string>
+#include <map>
+
+// Typedefs for tag hierarchy
+using NodeType =
```

Comment:
confusing use of the keyword "using" here. I have to remind myself to ask why.

Reply:
using is the "modern c++ typedef", I think it makes sense as it stands

Reply:
ok